library/src/main/java/com/crossbowffs/remotepreferences/RemotePreferenceProvider.java

```diff
@@ -76,20 +76,82 @@
      * authority and preference files. The authority must match the
      * {@code android:authorities} property defined in your manifest
      * file. Only the specified preference files will be accessible
-     * through the provider.
+     * through the provider. Kept for backward compatibility.
```

Comment:
I'd prefer you moved this doc to the "main" constructor below and replace this one with something like:

"See {@link (other constructor)}. All preference files are assumed to be located in credential protected (default) rather than device protected storage"